src/index.hbs

```diff
@@ -60,6 +60,15 @@
          * This can also be a function that returns the `on` string value.
          */
         on: 'ready',
+        /**
+         * If set, delays load of the external script until a set time after the `on` even fires.
+         * This setting is _not_ compatible with `initOnly`: If using `initOnly`, you must handle
+         * necessary delay before calling `load`.
```

Comment:
This statement isn't true :) the delay will still be applied to the manual `load` call